### PR TITLE
Apim 4864 2 title banner subtitle run e2e

### PR DESCRIPTION
**Pull request by illichso**
## Issue

https://gravitee.atlassian.net/browse/APIM-4864

## Description

For Portal Next: adding homepageTitle + homepageBannerTitle + homepageSubtitle

## Additional context

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-lzpihslodm.chromatic.com)

### CODE REVIEW

File: gravitee-apim-rest-api/gravitee-apim-rest-api-portal/gravitee-apim-rest-api-portal-rest/src/main/resources/portal-openapi.yaml [hunk 1]
```diff
@@ -4692,6 +4694,17 @@ components:
         homepageTitle:
           description: Main phrase to display on the homepage.
           type: string
+    ConfigurationPortalNext:
+      properties:
+        siteTitle:
+          description: Main phrase to display on the homepage.
```
Comment (jourdiw) [inline gravitee-apim-rest-api/gravitee-apim-rest-api-portal/gravitee-apim-rest-api-portal-rest/src/main/resources/portal-openapi.yaml]:
```suggestion
          description: Title to display in the navigation menu
```

File: gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/config/gravitee.yml [hunk 2]
```diff
@@ -660,3 +660,9 @@ api:
   properties:
     encryption:
       secret: vvLJ4Q8Khvv9tm2tIPdkGEdmgKUruAL6
+
+portal:
```
Comment (jourdiw) [inline gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/config/gravitee.yml]: We do not need to add this to the `gravitee.yml`. There should be documentation written by technical writers to explain how the user can add these settings to the `gravitee.yml` or in a cloud environment.

File: gravitee-apim-rest-api/gravitee-apim-rest-api-portal/gravitee-apim-rest-api-portal-rest/src/main/resources/portal-openapi.yaml (hunk 1 above)
Comment (illichso) [reply gravitee-apim-rest-api/gravitee-apim-rest-api-portal/gravitee-apim-rest-api-portal-rest/src/main/resources/portal-openapi.yaml]: Done

File: gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/config/gravitee.yml (hunk 2 above)
Comment (illichso) [reply gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/config/gravitee.yml]: Done

**Review by jourdiw — APPROVED** [review]

**Pull request merged**